assets/svg/success-green.svg

```diff
@@ -0,0 +1 @@
+<svg id="_1_Step_Darker" data-name="1 Step Darker" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 400 400"><defs><style>.cls-1,.cls-3,.cls-4,.cls-5,.cls-6,.cls-8{fill:none;}.cls-1{stroke:#e8eaed;stroke-linecap:round;}.cls-1,.cls-3,.cls-8{stroke-miterlimit:10;}.cls-1,.cls-8{stroke-width:6px;}.cls-2{fill:#f439a0;}.cls-3{stroke:#dadce0;stroke-width:7px;}.cls-4{stroke:#34a853;}.cls-4,.cls-5,.cls-6{stroke-linejoin:round;stroke-width:9px;}.cls-5{stroke:#188038;}.cls-6{stroke:#137333;}.cls-7{fill:#1e8e3e;}.cls-8{stroke:#fff;}</style></defs><line class="cls-1" x1="200" y1="10" x2="200" y2="41.42"/><line class="cls-1" x1="105" y1="35.46" x2="120.71" y2="62.66"/><line class="cls-1" x1="35.46" y1="105" x2="62.66" y2="120.71"/><line class="cls-1" x1="10" y1="200" x2="41.42" y2="200"/><line class="cls-1" x1="35.46" y1="295" x2="62.66" y2="279.29"/><line class="cls-1" x1="364.54" y1="295" x2="337.34" y2="279.29"/><line class="cls-1" x1="390" y1="200" x2="358.58" y2="200"/><line class="cls-1" x1="364.54" y1="105" x2="337.34" y2="120.71"/><line class="cls-1" x1="295" y1="35.46" x2="279.29" y2="62.66"/><path class="cls-2" d="M251.2,51.21c-.67-1,10.33-9.56,11.16-10.16,5.81-4.22,12.64-8,19.94-8.46,7.91-.53,15.45,2.58,23,4.58,10.31,2.77,21.44,3.8,32.09,1.69,9.59-1.9,20.83-5.9,27.87-12.93-.24.24.3,2.83.33,3.18a65.31,65.31,0,0,1,0,8.51,59.1,59.1,0,0,1-7,25.06,43.41,43.41,0,0,1-18.94,18.13C329.14,86,317.78,88.93,307,93.41c-6.7,2.79-18.11,7.23-18.11,16.07Z"/><line class="cls-3" x1="248.83" y1="47.55" x2="342.46" y2="192.24"/><path class="cls-4" d="M279.89,172.88c18.52,6.68,53.35,9.87,64.53-10.87,14.77-27.39-23.47-38.52-43.87-34.53"/><path class="cls-4" d="M88.94,145C13,233.15,70,296.93,119,267.62"/><path class="cls-5" d="M234.54,260.65c59.23,111,7.37,98-23.53,93.23l2,20.19"/><path class="cls-5" d="M186.58,274.6c31.93,144.76-28.4,65.93-35.58,53.33l-12.25,14.36"/><path class="cls-6" d="M234.54,260.65q3.92,7.32,7.19,14"/><path class="cls-6" d="M186.58,274.6q2.19,9.88,3.81,18.42"/><polygon class="cls-7" points="170.71 172.85 72.71 212.2 112.06 310.2 210.06 270.85 308.06 231.5 268.71 133.5 170.71 172.85"/><path class="cls-8" d="M158.45,268.63C142.2,271,125.93,263.43,114.7,250l83.92-22.8"/><line class="cls-3" x1="316.62" y1="152.31" x2="342.46" y2="192.24"/></svg>
```

Comment:
It's a bit odd that these all have the same `id` and `data-name` attributes. I think these should be removed as mentioned before but seems strange that all of these would have these in common.